docs/design/comparison-to-olm.md

```diff
@@ -45,12 +45,21 @@ The ODLM manages the deployment of the operands for OLM managed operators.  This
 ### uninstall
 
 - When users use **OLM** only, they need to delete every created custom resource and all the operators.
-- When using **ODLM**, if users don't need an operator application anymore, they can delete the OperandRequest created for the application. **ODLM** will handle the logic of checking if there are other users requesting this application and delete both custom resources and operators when they are not required.
+- When using **ODLM**, if users don't need an operator application anymore, they can delete the OperandRequest created for the application. **ODLM** will handle the logic of checking if there are other users requesting this application and delete both custom resources and operators when they are no longer referenced.
 
 ## Dependency Management
 
-- **OLM** will automatically create the dependent operator, which provides the CustomResourceDefinition or API that is required by the operator the user wants to install. The dependency management of **OLM** focuses on *API* level dependency, for example, the IAM operator depends on MongoDB operator CRD.
-- **ODLM** manages dependency by creating sub-OperandRequest to the creating the dependency operators and operands. The sub OperandRequest can be deployed by **ODLM** as an Operator custom resource or handled by operator logic.  The dependency management of **ODLM** focuses on *Application* level dependency, for example, IAM service depends on MongoDB service. Another gap in the **OLM** dependency management is operator and dependency operators must be in the same scope (OperatorGroup). Users can't make several operators from different scopes depends on a common operator. **ODLM** can handle these cases by creating OperandRequests.
+- **OLM** creates all required, dependent operators automatically by creating additional Subscriptions automatically.  Each operator statically defines all of it's REQUIRED dependencies in the form of CustomResourceDefinitions, API Service Definitions or Operator Package versions.  
+  - **OLM** operators declare which CRDs, and API Services are PROVIDED.
+  - **OLM** operators declare which CRDs, API Services and other Operator Package Versions are REQUIRED.
+  - All dependencies are statically defined as REQUIRED (vs. optional, or preferred), which result in all dependencies must be installed as a unit.
+  - All dependencies must be in the same OperatorGroup, which may require the operators to be in the same namespace. 
+  - Example:  The IAM Operator REQUIRES the Mongo DB CRD and the Mongo DB Operator PROVIDES the Mongo DB CRD.  When the IAM Operator Subscription is created, the Mongo DB Subscription is dynamically created, satisfying the Mongo DB CRD dependency.
```

Comment:
Can we not use the IAM and Mongo in examples?

Reply:
@pgodowski 

Do you mean we should use more general name, like `bar operator` and `foo operator`?

Reply:
@horis233 yes, perhaps a `parent` and `child` or something similar like requestor-requestee, etc

Reply:
Yeah, I agree... that would add some additional clarity.

Reply:
Changes in https://github.com/IBM/operand-deployment-lifecycle-manager/pull/634